scripts/github_api

```diff
@@ -71,6 +71,40 @@ def _print_output(output):
         print '%s="%s"' % (k, v)
 
 
+class Status:
+    """
+    https://developer.github.com/v3/repos/statuses/
+    """
+    def __init__(self, repo, reference):
```

Comment:
It makes more sense to me to have only a pull request param (from which you may infer repo and reference)